import-export-cli/integration/devFirst_test.go

```diff
@@ -38,73 +39,76 @@ func TestInitializeProject(t *testing.T) {
 		initFlag: projectName,
 	}
 
+	validateInitializeProject(t, args)
+	//Remove Created project and logout
```

Comment:
Can we move this _base.RemoveDir()_  into the _validateInitializeProject()_ function?